sigs.yaml

```diff
@@ -1669,9 +1669,15 @@ sigs:
   - name: reference-docs
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/reference-docs/master/OWNERS
+  - name: subproject-definition
+    owners:
+    - https://raw.githubusercontent.com/kubernetes/community/master/governance.md#subprojects
   - name: website
     owners:
     - https://raw.githubusercontent.com/kubernetes/website/master/OWNERS
+  - name: working-group-definition
+    owners:
+    - https://raw.githubusercontent.com/kubernetes/community/master/governance.md#working-groups
```

Comment:
This area has SIG Docs named subprojects: blog, localization, reference-docs, website
The links to the definition of subproject and WG does not belong here